src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,133 @@
+package banking.dao;
+
+import banking.configuration.DBManager;
+import banking.model.Account;
+import banking.model.Card;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class AccountDAO {
+
+    private static final String INSERT_INTO_ACCOUNT = "INSERT INTO account (number, pin) VALUES (?, ?)";
+    private static final String SELECT_COUNT_NUMBER = "SELECT count(number) FROM account WHERE number = (?)";
+    private static final String SELECT_ACCOUNT_WHERE_NUMBER_AND_PIN = "SELECT * FROM account WHERE number=? AND pin=?";
+    private static final String GET_ACCOUNT_WHERE_ID = "SELECT * FROM account WHERE id = ?";
```

Comment:
Can you use "BY" instead of "WHERE" in these constants?

Reply:
renamed